src/test/scala/de/zalando/beard/parser/BeardTemplateParserSpec.scala

```diff
@@ -7,492 +7,527 @@ import scala.io.Source
 
 class BeardTemplateParserSpec extends FunSpec with Matchers {
 
-  describe("when parsing an empty string") {
-    it("should return an empty BeardTemplate for an empty") {
-      BeardTemplateParser("") should be(BeardTemplate(List.empty))
+    describe("when parsing an empty string") {
```

Comment:
why is the formatting changed for this file? scala should be formatted with two spaces for new paragraphs...